README.md

```diff
@@ -7,60 +7,102 @@ BTCPayServer is an open-source, self-hosted payment processor for Bitcoin and ot
-
-* You want to be in control of your own funds
-* Compliance teams of traditional payment processors rejected your application
-* You want lower fees (we support SegWit)
-* You want a way to support payments in altcoins
-* You want to accept payments on the Lightning Network
-* You want universally compatible invoices (we support the Bitpay API)
-
 ![BTCPayInfographic](img/BTCPAYINFOGRAPHIC.png)
 
+## Features
+* Direct, peer-to-peer Bitcoin and altcoin payments
+* No transaction fees (other than those for the crypto networks)
+* No processing fees
+* No middleman
+* No KYC
+* User has complete control over private keys
+* Enhanced privacy
+* Enhanced security
+* Self-hosted
+* SegWit support
+* Lightning Network support (LND and c-lightning)
+* Altcoin support
+* Full compatibility with BitPay API (easy migration)
+* Process payments for others
+* Easy-embeddable Payment buttons
+* Point of sale app
+
+## How it works
+### In a nutshell
+In layman words, BTCPay Server is a self-hosted and automated invoicing system. When checking out, the customer is presented with an invoice. Each time, a different address is tied to a particular invoice, so there is no address re-use. BTCPay follows the status of the invoice through the blockchain and informs a merchant when he can fulfill the order safely.
+### How is it different
+BTCPay is a code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+### How it keeps funds secure
+Payments via BTCPay are direct, peer to peer. The merchant receives the coins directly to the wallet, with no intermediate. Securing your funds comes down to securing your private keys. There is an internal wallet inside BTCPay that merchants can use to spend the funds. The funds from that wallet can only be spent, if a transaction is signed with a compatible hardware wallet.
+### How it keeps data private
+The data is shared only between two parties - the buyer and a seller. Other payment processors share and collect user data with other parties, making it very hard to comply with privacy law regulations like GDPR.
+### How it resists censorship
+* Self-hosted
+* Can be run on a hardware
+* No central point of failure
+* Can easily be re-deployed
+
+BTCPay does not have a central point of failure since nobody is controlling it except for the user running it. If run on the cloud server, the hosting providers can potentially censor users by suspending hosting accounts or disabling access to virtual machines. Since no private keys are stored on the server, a censored individual can easily re-deploy the server with another host. Your coins are always inside your wallet. For ultimate censorship-resistance, users should run [BTCPay on their own hardware](HardwareDeployment.md).
```

Comment:
```suggestion
BTCPay does not have a central point of failure since nobody is controlling it except for the user running it. If run on the cloud server, the hosting providers can potentially censor users by suspending hosting accounts or disabling access to virtual machines. This is always a risk for anyone using a hosting provider. Since no private keys are stored on the server, a censored individual can easily re-deploy the server with another host. Your coins are always inside your wallet. If an invoice is paid while your BTCPay server is down, the software will automatically determine and notify the merchant of offline invoice payments when your server is back up. For ultimate censorship-resistance, users should run [BTCPay on their own hardware](HardwareDeployment.md).
```

Reply:
1. Let them know that host server suspension isn't related to BTCPay but rather centralized services
2. I think offline payments are always tracked with NBXplorer, right?